core.yaml

```diff
@@ -144,9 +144,6 @@ adminApps:
       - svc: argocd-server
         namespace: argocd
         type: public
-        auth: true
```

Comment:
I am not sure about it, because user will have login twice.
First to otomi console and then second to argocd

The idea behind oauth-proxy is that there is one auth cookie to cover access to all core apps.

Reply:
As we discussed this change is reverted now.